### ISSUE TITLE
Thrower inserters seem to copy more properties than one might deem intended. Also its impossible to repair a directed bounce pad.

### ISSUE THREAD

**Issue by Vet-zz-era** [Kiplacon/Renai-Transportation#28]
I probably shouldn't do 2 bug reports in a single post but.... Such is life.

When you do copy paste on an inserter if you say... Copy the properties of a long handed inserter to a stack inserter it converts the inserter into a long handed stack inserter. While this is a cool feature and would love to be able to keep it. It doesn't seem intentional. 

The other bug is that if I place a directed bounce pad and try to manually repair it. It seems to flash "Not Operable" like 20x/second and does not get healed. The strangest part of this bug is that it doesn't seem to happen to regular bounce pads or directed train bounce pads.

**Comment by Kiplacon** [comment]
That's certainly not intentional but I dont think I can fix it because that's just how Factorio copy and paste works. By how Factorio makes the inserters work, they can only either have the pickup and drop positions changeable or not changeable. I have to have them changeable because of the range adjusting, and there's no setting to only have the drop position changeable. So unfortunately I don't know much that can be done about it but I don't think it's too big a deal for it to be there.

For the bounce pad repair, the directed bounce pad is actually a reskinned constant combinator, unlike the regular bounce pad which is just a "block". I did that because at one point I wanted to be able to control which way the directed bounce pad was facing using the circuit network, or be able to bounce different items in different directions based on the signal, and it being a constant combinator under the hood would let you connect wires to it. But I never followed through on that idea and when I added the ability to toggle the bounce range indicators on the bounce pads on and off by clicking on them, the clicking was opening the constant combinator GUI which was annoying so I made it so all directed bounce pads couldn't be "operated" meaning it wouldn't open the GUI when you clicked it. I didn't know that it would prevent people from repairing it too though, that's new to me. I'll either change the directed bounce pads to a normal "block" entity or actually implement the direction control one day
